runtime/include/llvm/clang_builtins_wrapper.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Generated by clang_builtin_gen.py based on clang's Builtins.def */
+#define ADD_WRAPPER_PREFIX(S) chpl_clang_builtin_wrapper_##S
+static inline double ADD_WRAPPER_PREFIX(cimag)(_Complex double arg0)
```

Comment:
Since we're generating this file anyway, it seems an odd choice to use the macro to add the prefix. I think it'd be better to leave the generated file more straightforward and change that in the .py script if necessary.

Reply:
Macro is used here for consistency with with `chapel_libc_wrapper.h` file and how it's supposed to be used in future when we want to wrap more builtins.

Reply:
OK, I understand this point about consistency.  I will go ahead and allow this.